Sources/StructuredQueriesCore/Seeds.swift

```diff
@@ -67,7 +67,7 @@ public struct Seeds: Sequence {
   /// [SharingGRDB]: https://github.com/pointfreeco/sharing-grdb
   ///
   /// - Parameter build: A result builder closure that prepares statements to insert every built row.
-  public init(@InsertValuesBuilder<any Table> _ build: () -> [any Table]) {
+  public init(@SeedsBuilder _ build: () -> [any Table]) {
```

Comment:
The main gotcha in this PR is that we need a simultaneous release with SharingGRDB since it defines a `seeds` helper that will be incompatible with this feature.